### PR TITLE
ITEM.tsv New Lines

### PR DESCRIPTION
**Pull request by mamoruk**

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -8778,201 +8778,201 @@ ITEM_20151224_008777
...
+ITEM_20151224_008852	Collection: Gate Route
+ITEM_20151224_008853	Collection: Sirdgela Forest
+ITEM_20151224_008854	Collection: Kvailas Forest
+ITEM_20151224_008855	Collection: Watchman's Graveyard
+ITEM_20151224_008856	Collection: Gateway of the Great King
+ITEM_20151224_008857	Collection: Ramstis Ridge
+ITEM_20151224_008858	Collection: Overlong Bridge Valley
+ITEM_20151224_008859	Collection: Akmens Ridge
+ITEM_20151224_008860	Collection: Tiltas Valley
+ITEM_20151224_008861
```
Comment (ttgmichael) [inline ITEM.tsv]: I believe you are shifting a lot of lines down from these blanks.

ITEM_20151224_008861 콜렉션:기티스의 개척지 - > Collection: Gitis Settlement Area

**Comment by Sunaries** [comment]
- Put all the 'LV' in front of the name and change it to 'Lv'.
- Replace all instances of  'Can be used immediately after you have registered in the slot.' with 'Can be used after assigning it to a quick slot.'

**Comment by Sunaries** [comment]
Great job. Good to merge. :+1:

**Comment by imcgames** [comment]
Good Job @mamoruk and @Sunaries :+1:

**Pull request merged**